staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -311,9 +311,6 @@ func (o *DrainCmdOptions) RunDrain() error {
 				fmt.Fprintf(o.ErrOut, "error: unable to drain node %q due to error:%s, continuing command...\n", info.Name, err)
 				continue
 			}
-			fmt.Fprintf(o.ErrOut, "DEPRECATED WARNING: Aborting the drain command in a list of nodes will be deprecated.\n"+
-				"The new behavior will make the drain command go through all nodes even if one or more nodes failed during the drain.\n"+
-				"For now, users can try such experience via: --ignore-errors\n")
```

Comment:
If I am not wrong, we will make the --ignore-errors default, so no need to exist the flag and the validation in line 310?

Reply:
Yes, may be in next release ignore error will be the default.Then keep --ignore-errors flags or not, set this flag to false  which will terminated  drain action with error, i don't have prefer now.

Reply:
@yuzhiquan probably I misunderstood https://github.com/kubernetes/kubernetes/pull/98203#issuecomment-784664784 mention about removing the warning msg. I would keep such warning if we are adding new flag to change a behavior in 1.22.

/cc @eddiezane @rikatz WDYT ?